audit/projects/k8s-artifacts-prod-bak/buckets/eu.artifacts.k8s-artifacts-prod-bak.appspot.com/iam.json

```diff
@@ -29,5 +29,5 @@
       "role": "roles/storage.objectViewer"
     }
   ],
-  "etag": "CAo="
```

Comment:
These are super obnoxious.  If someone has time, we should add `jq` filtering to remove `etag` from these - `| jq 'del(.etag)'`  everywhere we see an etag

Reply:
Was also thinking about it. I'm gonna do it later

Reply:
@thockin created PR as a followup of this conversation: https://github.com/kubernetes/k8s.io/pull/688